ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/434__data-science-pipelines-ui.robot

```diff
@@ -64,7 +65,40 @@ Verify User Can Create, Run and Delete A DS Pipeline From DS Project Details Pag
     Delete Pipeline Run    ${PIPELINE_TEST_RUN_BASENAME}    ${PIPELINE_TEST_NAME}
     Delete Pipeline    ${PIPELINE_TEST_NAME}
     Delete Pipeline Server    ${PRJ_TITLE}
-    Delete Data Science Project    ${PRJ_TITLE}
+    [Teardown]    Delete Data Science Project    ${PRJ_TITLE}
+
+Verify Pipeline Metadata Pods Are Not Deployed When Running Pipelines
+    [Documentation]    Verifies that metadata pods are not created when running a data science pipeline,
+    ...         as this feature is currently disabled.
+    [Tags]    Smoke
+    ...       Tier1
```

Comment:
Isn't Tier1 == Sanity? 
Do you want this to be run in Smoke or Sanity?

Reply:
We have a lot of tests with Tier1 & Smoke tags, and others with Tier1 & Sanity. I think this is fine